### ISSUE TITLE
Marketshop

### ISSUE THREAD

**Issue by ecdarreola** [rathena/rathena#3267]
* **rAthena Hash**: e7fd6aa

* **Client Date**: 20180621a

* **Server Mode**: Renewal

* **Description of Issue**: Buying of items become weird if you choose random item in marketshop.
  * Result: The quantity you can add in the buying window is greater than the limit for the market shop. (See image below)
1st Picture:
![image](https://user-images.githubusercontent.com/5693492/42130411-f2dc5ba2-7d16-11e8-8a16-260ff13d833a.png)
Notice that in the buying items there's already 20pcs Turqoise added but in the selling items it appears to still have the 20 items.
2nd Picture:
![image](https://user-images.githubusercontent.com/5693492/42130416-1e82ee38-7d17-11e8-9b46-ea7b7213c287.png)
After buying, the items become gibberish or the bug happens.

This happens to the eden_market in the pull-request section which is status on-hold. Not sure if this happens also in normal marketshop npc.

See image below for other reference with wrong update of quantity.
![image](https://user-images.githubusercontent.com/5693492/42130452-063e26e8-7d18-11e8-8808-03c9eb3ff315.png)

 
  * Expected Result: Quantity should be updated in the selling items window real-time and after buying there should be no gibberish text.
  * How to Reproduce: 
1. Use Eden_Market Pull Request (Status On-Hold) - For simulation purposes
2. Go to Aurora since it is the NPC that I tested
3. Repeat process above shown (the picture)
4. Bug Happens
  * Official Information: I don't have official information
  

* **Modifications that may affect results**: PR EP16, PR Arch_Bishop, PR Logging, PR Nightmare

**Comment by cydh** [comment]
I believe this is client related. Invalid in 37330448586ad909de7feae75c5e507646ca6619 with 2015-11-04a
people nowadays really like to race and use unsupported client without giving the client info :-1: 
come back after u can test with supported client to with client packet list to test

**Comment by ecdarreola** [comment]
I used 20180621a, since as per update before we can use 2018 client onwards as long as kRO doesn't add encryption packets since almost packets are the same.

The client info was given stating it was 2018-06-21a.

**Comment by cydh** [comment]
that's why I said so

**Comment by ecdarreola** [comment]
Yeah but talking about "unsupported clients" I think the statement is invalid. Since 2018 clients at the first place are supported by default as long as there is no changes with regards to encryption. I just can't find the thread about it but was pretty sure it was said 2018 clients are supported.

So yeah, if the status is won't fix might as well close this one.

**Comment by NordGate** [comment]
Also on 2018-06-20e which is supposed to be "the last supported client", yet you guys just keep closing the tickets.
https://rathena.org/board/topic/117168-packetver-20180620-client-release-2018-06-20eragexere/

fix the bugs instead of just rolling the blame on users.

**Comment by Lemongrass3110** [comment]
I dont know why you think that this is the "last supported client", when the default packet version is 2020-04-01.
So maybe instead of putting the blame of a broken client provided by Gravity (which was fixed later on again and this is not the first and definitely not the last client where such a thing happened) on us you could consider upgrading your client.
